### PR TITLE
Removed extra validation fields from user validator

### PR DESCRIPTION
**Pull request by TD-17**
issue  #246

### CODE REVIEW

File: middlewares/validators/user.js
Comment (swarajpure) [inline middlewares/validators/user.js:23]: I just realised we're sending `website` from the signup form, I'll change the API contract, please remove `site` and keep `website`

**Comment by swarajpure** [comment]
Need to have clarification from the frontend engineer who worked on the signup form as what key are we using for the company name of the user. I highly doubt it's `company_name`. Some users in the DB have got `company` while some have got `company_name`

If we make any changes to the validators, the signup form will stop working which is not expected at this point of time. Let's keep it on hold till then.

**Comment by TD-17** [comment]
> Need to have clarification from the frontend engineer who worked on the signup form as what key are we using for the company name of the user. I highly doubt it's `company_name`. Some users in the DB have got `company` while some have got `company_name`
> 
> If we make any changes to the validators, the signup form will stop working which is not expected at this point of time. Let's keep it on hold till then.

yeah okay!

File: middlewares/validators/user.js
Comment (ankurnarkhede) [inline middlewares/validators/user.js:21]: @swarajpure , we won't actually be storing tokens via this route, right?
So this validation can be removed.

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]

File: middlewares/validators/user.js
Comment (swarajpure) [reply middlewares/validators/user.js:21]: Yes, that can be removed as well

File: middlewares/validators/user.js
Comment (swarajpure) [reply middlewares/validators/user.js:23]: @TD-17

File: middlewares/validators/user.js
Comment (swarajpure) [reply middlewares/validators/user.js:21]: @TD-17

File: middlewares/validators/user.js
Comment (swarajpure) [reply middlewares/validators/user.js:23]: Let's remove `company_name`

Comment (TD-17) [reply middlewares/validators/user.js:23]: yeah okay

File: middlewares/validators/user.js
Comment (swarajpure) [inline middlewares/validators/user.js:20]: We don't need this here -- `isMember`
It should not be here from security pov
Otherwise users will mark themselves as members 🤣 

Please remove it from here

Comment (TD-17) [reply middlewares/validators/user.js:20]: oh yeahh! Okay

**Review by swarajpure — CHANGES_REQUESTED** [review]
Can we also remove `github_display_name` and `github_id` since this is data coming from github, we shouldn't allow users to change it themselves.

This is last change, after which we can merge this one :P

**Comment by TD-17** [comment]
> Can we also remove `github_display_name` and `github_id` since this is data coming from github, we shouldn't allow users to change it themselves.
> 
> This is last change, after which we can merge this one :P

yess okay :)

**Review by swarajpure — APPROVED** [review]
Looks good to me!
Thanks! 

Let's wait for at least one more reviewer to have a look here as 1 mistake here can cause a big problem somewhere. Let's not merge till then.

**Comment by TD-17** [comment]
> Looks good to me!
> Thanks!
> 
> Let's wait for at least one more reviewer to have a look here as 1 mistake here can cause a big problem somewhere. Let's not merge till then.

yeah cool

**Review by sumitd94 — APPROVED** [review]

**Pull request merged**
